ui/components/logs/log-detail-sheet.tsx

```
@@ -0,0 +1,196 @@
+							maxHeight={450}
+							wrap={true}
+							code={toolsParameter}
+							lang="json"
+							readonly={true}
+							options={{ scrollBeyondLastLine: false, collapsibleBlocks: true, lineNumbers: "off", alwaysConsumeMouseWheel: false }}
+						/>
+					</div>
+				)}
+
+				<div className="mt-4 w-full text-center text-sm font-medium">Conversation History</div>
+				{log.input_history && log.input_history.map((message) => <LogMessageView message={message} />)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)